src/lib.rs

```diff
@@ -561,6 +561,23 @@ impl Ciphertext {
 #[derive(Clone, Deserialize, Serialize, PartialEq, Eq)]
 pub struct DecryptionShare(#[serde(with = "serde_impl::projective")] G1Projective);
 
+impl DecryptionShare {
+    /// Serializes the decryption share into a byte vector.
+    pub fn to_bytes(&self) -> [u8; 48] {
+        self.0.to_affine().to_compressed()
+    }
+
+    // FIXME: make CT
```

Comment:
What does CT mean?

Is this covered by a test?

Reply:
* CT = constant time, that would mean returning a `CtOption`, which is totally possible, I just remember it to be slightly annoying and isn't really necessary for encoding essentially public data.
* `Encodable`/`Decodable` are defined in `fedimint_core`, so not available here. https://github.com/fedimint/fedimint/pull/2333 uses this PR to impl them in `fedimint_core`.
* No, let me fix this.

Reply:
Added a test